pkg/logs/internal/tailers/traps/tailer_test.go

```diff
@@ -64,11 +65,15 @@ func TestTrapsShouldReceiveMessages(t *testing.T) {
 	formattedPacket := format(t, p)
 	assert.Equal(t, message.StatusInfo, msg.GetStatus())
 	assert.Equal(t, formattedPacket, msg.Content)
-	assert.Equal(t, []string{
-		"snmp_version:2",
+	expectedTags := []string{
 		"device_namespace:default",
 		"snmp_device:127.0.0.1",
-	}, msg.Origin.Tags())
+		"snmp_version:2",
+	}
+	tags := msg.Origin.Tags()
+	sort.Strings(expectedTags)
+	sort.Strings(tags)
+	assert.Equal(t, expectedTags, tags)
```

Comment:
You might also use `assert.ElementsMatch(t, []string{ ... }, tags)`

Reply:
✅